<launch>

    <arg name="fuse_odom" default="true" />
    <arg name="record" default="true" />


    # hardware
    <include file="$(find hardware)/hardware.launch" >
    </include>
    
    # localization
    <include file="$(find localization)/localization.launch" >
    </include>

    # ramp steer test 1
    <node name="sine_steer" pkg="sysid" type="sine_steer_exp1" required="true" output="screen">
        <remap from="odom_fused" to="odom" unless="$(arg fuse_odom)" />
    </node>

	# Rosbag data collection
    <node pkg="rosbag" type="record" name="rosbag_sine_steer_exp1"
        args="record -o /home/nvidia/Documents/rosbags/sysid/sine_steer_exp1_trial0_ /ekf_localization/odom /commands/keyboard /odom_fused /imu /sensors/servo_position_command"
        if="$(arg record)" />

</launch>
